### ISSUE TITLE
Settings for Widescreen mode using Retroarch Plus on Android

### ISSUE THREAD

**Issue by rdsngit** [DerKoun/bsnes-hd#86]
@DerKoun In the Patreon comments discussion for the Super Mario World Widescreen beta v01 you explained that sometimes Android users of the bsnes-hd core in Retroarch Plus found `.bso` files were not overriding the local core settings. Would it be useful to add a section to the bsnes-hd Readme to explain to Android users which settings need to be used for Widescreen?

For context, I had the patched `.sfc` file in the same directory as the SMW `.bso` file. It initially displayed in regular SNES aspect ratio, eg 4:3. Then I changed some of the Settings that I though were related to get 16:9 Widescreen mode running successfully.

I've included a set of screenshots showing all the Settings, here are the ones I think could have been key to getting Widescreen working

```
Widescreen Mode = all
Widescreen Aspect Ratio = 16:9
Widescreen Sprites = unsafe
Pixel Aspect Correction = Off
```

Before                  |  After
:----------------:|:-----------------------:
![widescreen-not-working-smw-bsnes-hd-android-retroarch-plus](https://user-images.githubusercontent.com/52106678/122076439-ca046f00-cdf2-11eb-9706-27873c723d47.png) | ![widescreen-smw-bsnes-hd-android-retroarch-plus](https://user-images.githubusercontent.com/52106678/122076474-d092e680-cdf2-11eb-9747-3dc678a9f0fa.png)

Settings part 1                    |  Settings Part 2             | Settings Part 3
:-------------------------:|:----------------------:|:-----------------------:
![part1of3-widescreen-settings-bsnes-hd-android-retroarch-plus](https://user-images.githubusercontent.com/52106678/122076580-e43e4d00-cdf2-11eb-9b07-bce9637fcd7b.png) | ![part2of3-widescreen-settings-bsnes-hd-android-retroarch-plus](https://user-images.githubusercontent.com/52106678/122076601-e9030100-cdf2-11eb-9a9d-f9beec52eb26.png) | ![part3of3-widescreen-settings-bsnes-hd-android-retroarch-plus](https://user-images.githubusercontent.com/52106678/122076610-ebfdf180-cdf2-11eb-8b18-eaf9f77f4f82.png)

**Comment by DerKoun** [comment]
I hope I can fix the BSO issue. But I'll talk to Vitor about including the settings (they vary between different games/patches). For that I'll have to add at least one BSO-exclusive setting to the core option, which I have planned for the next release as well.

**Comment by rdsngit** [comment]
Sounds like you have some ideas for fixing this for future releases. Could it be useful to have a bsnes-hd core setting to tell the core to look for and use the BSO file if it is present? Eg something like "Use BSO file for Settings = true / false"

**Comment by DerKoun** [comment]
Not sure in which use-case disabling the BSO would be useful. But it is possible.

**Comment by rdsngit** [comment]
Ok, if you're planning to fix that so it detects and applies BSO automatically then that sounds a good approach. Should we close this issue if you're already working on this for a future release?

**Comment by DerKoun** [comment]
I'll close it after the release.

**Comment by DerKoun** [comment]
Please try the BSO again in the latest version (10.6) and let me know if it works. If it doesn't first please tripple check if you named the 3 files correctly. Then, if possible, try the RetroArch APK instead of the Play Store version. Thanks

**Comment by rdsngit** [comment]
Hi Derkoun - thanks for the update. I saw on Twitter libretro posted that retroarch was working with SMW Widescreen. I give this a try today and let you know how it works.

**Comment by rdsngit** [comment]
@DerKoun Thanks for the update - I can confirm this is now working.

Tested this today using the bsnes-hd version 10.6 using the latest version of Retroarch Plus for Android. 

Steps:
- Reset the bsnes-hd settings back to their defaults.
- Added the SMW Widescreen v1.2 patch and BSO file with an unpatched SMW rom to the same directory
- Renamed all three files names to the same name in the same directory
- Started the core and selected the SFC file
- It loaded into 16:9 widescreen mode

This worked in 16:9 widescreen mode. There were some related things about aspect rations that I found. If you go into the core settings and change Widescreen Aspect Ratio to 21:9 it stretches the 16:9 game display area and 16:10 squashes the game display area. I assume these are left in for users that have different size displays and want to user the SMW Widescreen patch.
